roxie/ccd/ccdmain.cpp

```diff
@@ -723,7 +723,7 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
 #ifdef _CONTAINERIZED
         getDefaultStoragePlane(defaultPlane);
         {
-            Owned<IStoragePlane> plane = getStoragePlane(defaultPlane, true);
+            Owned<IStoragePlane> plane = getDataStoragePlane(defaultPlane, true);
```

Comment:
Not new, but the fact that this throws an exception when not set is a bit of a pain for standalone executable usage. Probably only affects me though.